examples/get_cli_test_data.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+import json
+import os
+
+import requests
+
+from starfish import Experiment
+from starfish.util.argparse import FsExistsType
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("experiment_url")
+    parser.add_argument("output_dir", type=FsExistsType())
+    args = parser.parse_args()
+
+    # save image stacks locally
```

Comment:
So I ended up doing this kinda hackily to avoid needing those zip files. I figured since this is just for these cli tests this might be ok but @ttung let me know if you want a more long term solution for saving an experiment object to a direcotry.

Reply:
This is almost perfect.

I would just change it to:

```
for fov in exp.fovs():
    os.mkdir(fov.name)
    fov.write(....)
```